app/build.gradle

```diff
@@ -58,32 +58,32 @@ android {
 }
 
 dependencies {
-    api project(':uxkit')
```

Comment:
we prefer implementation over api so that apps can have access to the underlying API, classes, views etc...

Reply:
It's an sample app module which is not used as library and does not need transitive dependencies.

Reply:
there are api -> implementation in various other places

Reply:
Because some of them should not be transitive. Developer using uxkit library doesn't care about Picasso library, he's interested only in DefaultIncomingCallActivity. If you have some specific dependency in mind please add another comment on it.